docs/oauth2-oidc/custom-login-consent/flow.mdx

```diff
@@ -3,61 +3,55 @@ id: flow
+instead uses HTTP redirection to "delegate" the login flow to another app - this is the "Ory OAuth 2.0 login & consent flow".
 
 ## Example implementation
 
-You can find an example [Node.js implementation on Ory's GitHub](https://github.com/ory/hydra-login-consent-node) for an app that
-implements both custom login and consent.
+You can find an example
+[Node.js implementation of an app that implements custom login and consent on Ory's GitHub](https://github.com/ory/hydra-login-consent-node).
 
 ## Login
 
-The OAuth2 2.0 / OpenID Connect Flow is initiated by pointing the End-User's browser to the `/oauth2/auth` endpoint. Depending on
-which flow ("Authorize Code Flow", "Implicit Flow", ...) you want to use, some of the query parameters (for example
-`/oauth2/auth?response_type=code`, `/oauth2/auth?response_type=token, ...) might change but the overall initiation works always by
-sending the browser to that URL.
+The OAuth2 2.0 / OpenID Connect Flow is initiated by pointing the end user's browser to the `/oauth2/auth` endpoint. Depending on
+the OAuth2 flow you want to use, some of the query parameters (for example `/oauth2/auth?response_type=code` or
+`/oauth2/auth?response_type=token`) can change. Starting flows always involves sending the browser to that URL.
 
-The next task for Ory OAuth2 & OpenID Connect is to know the user of the request. To achieve that, Ory OAuth2 & OpenID Connect
+The next task for Ory OAuth2 and OpenID Connect is to know the user of the request. To achieve that, Ory OAuth2 and OpenID Connect
 checks if a session cookie is set containing information about a previously successful login. Additionally, OpenID Connect
-parameters id_token_hint, prompt, and max_age are evaluated and processed. Depending on their values and the login state, the user
-might need to re-authenticate or the flow will fail.
+parameters `id_token_hint`, `prompt`, and `max_age` are evaluated and processed. Depending on their values and the login state,
+the user might need to re-authenticate or the flow will fail.
 
-To authenticate the user (this happens regardless of whether a session exists for the user or not), Ory OAuth2 & OpenID Connect
-redirects the browser to the "Login Endpoint" established in your config:
+To authenticate the user (this happens regardless of whether a session exists for the user or not), Ory OAuth2 and OpenID Connect
+redirects the browser to the "login endpoint" established in your config:
 
 ```shell
 ory patch oauth2-config
   --replace "/urls/login=\"https://login.example.org/oauth2-login\""
 ```
 
-Ory OAuth2 & OpenID Connect appends a login_challenge query parameter to the URL. The value is an ID which should later be used by
-the Login Endpoint to fetch important information about the request.
+Ory OAuth2 and OpenID Connect appends a `login_challenge` query parameter to the URL. The value is an ID which should later be
+used by the login endpint to fetch important information about the request.
```

Comment:
```suggestion
used by the login endpoint to fetch important information about the request.
```

"endpint" makes me think of a glass of Guinness after work 😀

Reply:
The endpint is just the beginning 🍺 🍻 🍺